server/server.go

```diff
@@ -11,11 +11,23 @@ import (
 	"github.com/src-d/lookout/store/models"
 	"github.com/src-d/lookout/util/ctxlog"
 	"github.com/src-d/lookout/util/grpchelper"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"gopkg.in/src-d/lookout-sdk.v0/pb"
 
 	log "gopkg.in/src-d/go-log.v1"
 	yaml "gopkg.in/yaml.v2"
 )
 
+var grpcErrorMessages = map[lookout.EventType]map[codes.Code]string{
+	pb.PushEventType: map[codes.Code]string{
+		codes.DeadlineExceeded: "timeout exceeded, try increasing analyzerPushTimeout",
```

Comment:
`analyzerPushTimeout` is an internal name that will not be familiar to a user. Let's change this instead to 
```suggestion
		codes.DeadlineExceeded: "timeout exceeded, try increasing analyzer_push in config.yml",
```

Same for the other message below.

Reply:
Addressed in [here](https://github.com/src-d/lookout/pull/442/commits/106b581bd65736c49b75b74f9218bf63ccf26ec7)